.github/workflows/build.yaml

```
@@ -15,7 +15,7 @@ jobs:
         strategy:
             matrix:
                 php-version:
-                    - "8.1"
+                    - "8.2"
```

I don't know why the current version is  `8.1`. I usually have the coding standards set at the lowest allowed PHP version (in this case `7.4`) to avoid any syntax errors being made with coding standards fix.

TL;DR: Let's keep the `8.1` for this step

```suggestion
                    - "8.1"
```